src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/IconBitmapStatusUtility.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.PackageManagement.UI
+{
+    internal static class IconBitmapStatusUtility
+    {
+        internal static bool GetIsDefaultIcon(IconBitmapStatus bitmapStatus)
```

Comment:
```suggestion
        internal static bool IsDefaultIcon(IconBitmapStatus bitmapStatus)
```

Reply:
gonna keep the "Get" prefix due to: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-methods